### ISSUE TITLE
Project status?

### ISSUE THREAD

**Issue by mhauru** [chaoss/augur#2766]
Our organisation is looking for a tool to collect data from GitHub and analyse the status of our open source repos and the contributions we make to the ecosystem. I've been trying to set up Augur over the past few days to see if it would serve our needs. It seems quite a lot has changed recently, with v0.63.0. The documentation seems somewhat out of date and I ran into a few bits in the code that seemed half-finished, that I needed to comment out or otherwise fix before I got the current main to run.

Could I please ask someone to summarise the current state of the project for me? Is it currently in a state where one can clone main and start using Augur, or should I rather clone some older version tag and find a stable version there, or should I come back once the current ongoing work is finished?

**Comment by sgoggins** [comment]
Hi @mhauru : It is hard to know what problems you encountered without specific traces or screen shots of what failed. CHAOSS software is not easily installed, though we do continue to make improvements and many organizations have successfully installed and run Augur for their benefit. 

Please let us know if you have specific errors we should look into. Information about the operating system, python version, etc. as suggested in our Bug Template will be helpful. Thank you for giving Augur a whirl and please open an issue with enough technical detail for us to address! :) Again, thank you!!

**Comment by mhauru** [comment]
Thanks for the response @sgoggins! I agree and understand that my above message doesn't really qualify as a bug report. What I was looking for here was to ask about the stability and status of the project more broadly. Correct me if I got this wrong, but it sounds like the main branch is considered stable and ready to be used, and any issues on it should be reported. I'll open some more specific issues for the things I ran into.
